### PR TITLE
Add --- at the beggining of *.y*ml files

### PR DESCRIPTION
**Pull request by StanIwan**
## What does this PR do?
https://phorge.wireload.net/T7291

Initially, I thought the task to be about implementing StrictYAML parser in the project (based on this issue https://github.com/Screenly/cli/issues/61), @sergey-borovkov clarified that it's just a simple fix of the issues that `yamllint` shows.

### CODE REVIEW

**Comment by StanIwan** [comment]
@sergey-borovkov, should I also update files in `.github` repository?

image

**Comment by StanIwan** [comment]
@sergey-borovkov running `yamllint` on the `cli` projects reports truthy warning, but it seems that it's not right, should I add a `.yamllint` config file to the project that would suppress it?

image

**Comment by sergey-borovkov** [comment]
I am very confused here. GH Actions don't really matter...
That's the function that needs to be fixed. 
https://github.com/Screenly/cli/blob/a29bdacfbdf10be82d942f3bff9609f186885a29/src/commands/mod.rs#L267

File: action.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # action.yml
+---
```
Comment (sergey-borovkov) [inline action.yml:2]: no need.

**Pull request closed without merging**